library/catalog/templates/collections.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}Collections{% endblock %}
+
+{% block body %}
+
+<div class="container" style="min-height: 660px">
+
+    <h2 style="margin-top: 25px; margin-bottom: 20px;">Books with over 1000 pages</h2>
+    <div class="swiper-container first-container">
+      <!-- Additional required wrapper -->
+      <div class="swiper-wrapper">
+        <!-- Slides -->
+          {% for book in pages_books %}
+            <div class="swiper-slide" style="position: relative; margin-bottom: 10px;">
+                <a href="{% url 'book-details' book.id %}">
+                    <img src="{{ book.link_img }}" alt="{{ book.title }}">
+                    <p><b>{{ book.title }}</b></p>
+                    <p class="text-bottom">Pages: {{ book.pages }}</p>
+                </a>
+            </div>
+          {% endfor %}
+      </div>
+    </div>
+    <div class="swiper-button-next first-next" style="position: relative; top: -278px; left: 1150px"></div>
```

Comment:
Move inline CSS to the separate file.

Reply:
I moved CSS to a separate file, but I used !important statement due to priority